build.py

```diff
@@ -522,6 +522,17 @@ def create_dockerfile_buildbase(ddir, dockerfile_name, argmap, backends):
     pip3 install --upgrade wheel setuptools docker && \
     pip3 install grpcio-tools grpcio-channelz
 
+# Install DCGM
```

Comment:
This install is needed in buildbase container so that we can build tritonserver. You should move it down with the other related code so that it is all together. The install is also needed in create_dockerfile_linux() as that is the dockerfile that builds the actual production container. Move it into a separate function since it needs to be used in 2 places.

Reply:
Since we added in the buildbase would that information not be included in the linux version?

Reply:
No, look at create_dockerfile_linux and you can see that we start from the min container and just copy some things over from the build container. We do that because we don't want the production container to contain all the build dependencies and artifacts. That also means that we need to explicitly install any dependencies that are needed to run tritonserver, as you can see us doing in create_dockerfile_linux(). In general the production tritonserver container is not based on the build containers... it just copies out the built libraries and executables from the build container.